infrastructure/main.tf

```diff
@@ -110,3 +114,29 @@ data "azurerm_key_vault_secret" "oauth2_secret" {
 provider "azurerm" {
     version = "1.22.1"
 }
+
+data "azurerm_subnet" "core_infra_redis_subnet" {
+  name                 = "core-infra-subnet-1-${var.env}"
+  virtual_network_name = "core-infra-vnet-${var.env}"
+  resource_group_name  = "core-infra-${var.env}"
+}
+
+data "azurerm_key_vault_secret" "redis_secret" {
+  name      = "redis-secret"
+  key_vault_id = "${data.azurerm_key_vault.div_key_vault.id}"
+}
+
+resource "azurerm_key_vault_secret" "redis_connection_string" {
+  name = "${var.component}-redis-connection-string"
+  value = "redis://ignore:${urlencode(module.redis-cache.access_key)}@${module.redis-cache.host_name}:${module.redis-cache.redis_port}?tls=true"
+  key_vault_id = "${data.azurerm_key_vault.div_key_vault.id}"
```

Comment:
some references to div, want to fix them ?

Reply:
cool, i will changed. thx for spotting it

Reply:
changed